paimon_python_java/paimon-python-java-bridge/tools/ci/paimon-ci-tools/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can just use https://mvnrepository.com/artifact/org.apache.paimon/paimon-ci-tools/0.9.0